site3/website/scripts/javadoc-gen.sh

```diff
@@ -34,7 +34,7 @@ function build_javadoc() {
   if [[ "$use_gradle" == "true" ]]; then
     ./gradlew generateApiJavadoc
   else
-    mvn clean install javadoc:aggregate -DskipTests
+    mvn clean -B -nsu -am -pl :bookkeeper-common,:bookkeeper-server,:bookkeeper-stats-api,:bookkeeper-stats-providers,:codahale-metrics-provider,:prometheus-metrics-provider install javadoc:aggregate -DskipTests -Pdelombok
```

Comment:
the Maven standard syntax for -pl does not require the colon, I'd prefer to remove them